rcsbapi/data/data_schema.py

```diff
@@ -209,77 +209,94 @@ def _construct_root_dict(self) -> Dict[str, List[Dict[str, str]]]:
                 root_dict[root_name].append({"name": arg_name, "description": arg_description, "kind": arg_kind, "type": arg_type})
         return root_dict
 
-    def fetch_schema(self) -> Dict:
+    def _fetch_schema(self) -> Tuple:
         """Make an introspection query to get full Data API query.
         Can also be found in resources folder as "data_api_schema.json"
 
         Returns:
-            Dict: JSON response of introspection request
+            Tuple: Tuple of introspection query string and JSON response of introspection request
         """
```

Comment:
This should be returning just one thing now, right? Not a tuple?

Reply:
yes, sorry, I forgot to update the docstring. I just pushed the correction